cmd/gaia/testnets/README.md

```diff
@@ -159,70 +98,70 @@ Start the full node:
 gaiad start
 ```
 
-Check the everything is running smoothly:
+Check that everything is running smoothly:
 
 ```
 gaiacli status
 ```
 
-## Generate keys
+View the status of the network with the [Cosmos Explorer](https://explorecosmos.network).Once your full node syncs up to the current block height, you should see it appear on the [list of full nodes](https://explorecosmos.network/validators).
```

Comment:
I thinks it's hard to see your node in this explorer because the number of connections is limited